lib/ace/ext/papeeria/katex-previewer.coffee

```diff
@@ -34,6 +35,10 @@ define((require, exports, module) ->
     }
 
     initKaTeX = (onLoaded) ->
+      # Loading banned sequences file
+      response = $.getJSON(require.toUrl("./banned_token_sequences.json"))
```

Comment:
The list is kinda big, and it will supposedly get bigger in the future. I thought it would be much more convenient to keep it in a separate file.